### DISCUSSION TITLE
Code For The World - Concept

### DISCUSSION THREAD

**Discussion by ThaZeno** [community/community#37039] [Other Feature Feedback, Questions, & Ideas]
Here's the idea:

• An integrated time counter for the GitHub built-in web editor. (Or just a counter of how much time was spent on GitHub). Basically, time spent coding 

• For every hour of "coding" done in the world, $0.01 (or another amount) is added to a monthly "bank". The company or organization that would be the generous donor of all these singular cents has yet to be defined. 

• Then, every month, the entire community votes on what charity or organization to give the money to. 

I don't know if this has been done before, and yes it might be far-fetched as has yet to become concrete,  but in any case, it seems like a good idea.

**Comment by ossanna16** [comment]
Hi there @ThaZeno and welcome to our community 😄. Thanks so much for the suggestion!

**Reply by ThaZeno** [reply]
Thanks!

**Reply by HanGPIErr** [reply]
> Here's the idea:
> 
> • An integrated time counter for the GitHub built-in web editor. (Or just a counter of how much time was spent on GitHub). Basically, time spent coding
> 
> • For every hour of "coding" done in the world, $0.01 (or another amount) is added to a monthly "bank". The company or organization that would be the generous donor of all these singular cents has yet to be defined.
> 
> • Then, every month, the entire community votes on what charity or organization to give the money to.
> 
> I don't know if this has been done before, and yes it might be far-fetched as has yet to become concrete, but in any case, it seems like a good idea.

That's not a bad idea.

Coding a lot, and get rewarded for a charity organization of association, could be a great idea, like plant a tree or similar idea at each goal reach.

**Comment by mariadhari6** [comment]
wow

**Comment by mariadhari6** [comment]
@ThaZeno  I think, that's great idea

**Reply by ThaZeno** [reply]
Thanks so much! Still trying to get more input so we can get it started
